### ISSUE TITLE
Demo link stale

### ISSUE THREAD

**Issue by samuelhackwill** [aldeed/meteor-autoform#1711]
Demo link [http://autoform.meteorapp.com](http://autoform.meteorapp.com) seems dead :  

> 404 Not Found: No applications registered for host 'autoform.meteorapp.com'.

**Comment by jankapunkt** [comment]
Yes we should either use the free tier hosting or create a client only Meteor app and host via GitHub Pages

**Comment by samuelhackwill** [comment]
interesting, i wasn't aware it was possible to make a client-only meteor app, sounds nice.

https://forums.meteor.com/t/any-way-to-make-a-100-client-side-app-thats-only-html-js/4940
